### ISSUE TITLE
Add a way to add specific songs in playlist

### ISSUE THREAD

**Issue by LittleWhite-tb** [EragonJ/Kaku#494]
I would like to be able to navigate into my playlists and to add one song in the playlist, then go in another playlist and add another song and so on.
Same for the search results. Like this, I would be able to build a temporary playlist fitting my mood.

How I see the feature : just add an new entry in the right click of a song, to let the user to add it in the playlist.

**Comment by EragonJ** [comment]
I think what you need is more like a general queue ?

**Comment by LittleWhite-tb** [comment]
Yes, a queue that I can fill just for this Kaku session in order to have around several hours of playing without searching new song each time a song does finish
